src/button/src/components/styled/DefaultButton.js

```diff
@@ -16,7 +16,7 @@ const defaultStyle = css`
   color: ${NEUTRALS[1]};
   cursor: pointer;
   font-size: 14px;
-  line-height: 18px;
+  line-height: inherit;
```

Comment:
At first glance, I'm quite opposed to `inherit`. What's the use case for inheriting `line-height`? I think `normal` would make more sense, but I'd still push back.

The reason I specifically set `line-height: 18px` as opposed to using `normal` is because it gives the overall button a nice even height at `40px`. With `line-height: normal`, the overall height is `39px`. With `line-height: inherit`, the overall height is unpredictable. I much prefer even heights / widths over odd.

I think a better approach would be to expose a `height` prop on the button, and set the various heights internally.

Reply:
By default, setting `line-height: normal` will be roughly equal to 1.20 based on whatever browser is using it. Paul's style guide calls for using 1.35, which places the overall height at 40px. Using inherit will also ensure that it will be uniform through out the app and manual adjustments based on the font size is not necessary.

Reply:
Basic principle: the button height should always be an even number. With that in mind, I don't see how you can guarantee the height to be even with `line-height: inherit`. For example:

```
<div style={{ lineHeight: '19px' }}>
  <button />
</div>
```

Here, the parent sets the line height to `19px`, which means the our default button would get an odd height... 10px padding top + 10px padding bottom + 1px border top + 1px border bottom + 19px height = 41px.

Reply:
I see. In that case I feel it would be more prudent to explicitly define the height of the button, as opposed to using line-height as a proxy.